src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -183,4 +183,18 @@ static FabricLoader getInstance() {
 
 	@Deprecated
 	File getConfigDirectory();
+
+	/**
+	 * Gets the command line arguments used to launch the game. If this is printed for debugging, make sure {@code sanitize} is {@code true}.
+	 * @param sanitize Whether to remove sensitive information
+	 * @return the launch arguments
+	 */
+	String[] getLaunchArguments(boolean sanitize);
+
+	/**
+	 * @see FabricLoader#getLaunchArguments(boolean)
+	 */
+	default String[] getLaunchArguments() {
```

Comment:
I'd probably mention that this does not return any sensitive information.

Reply:
Done!